src/test/java/org/cactoos/scalar/AvgOfTest.java

```diff
@@ -178,4 +178,86 @@ public void withScalars() {
         );
     }
 
+    @Test
+    public void withMaxValueCollections() {
```

Comment:
@allanws I think interesting include more tests, like `100.0` and `100.266` as suggested by @svendiedrichsen, and `Double.MAX_VALUE` and `0`, the last three and four `Double.MAX_VALUE` numbers (`Double.MAX_VALUE`,  `Double.MAX_VALUE - 1` and `Double.MAX_VALUE - 2` for example.